scripts/build.sh

```
@@ -887,298 +1058,328 @@ run_component_build "systemd" build_systemd_component
+  if should_run_target "all" && [ "${TARGET_RESULTS[all]:-}" = "success" ]; then
+    # Create a minimal LSB root filesystem image
+    lsb_img="$ARTIFACTS_DIR/images/lsb_root.img"
+    rm -f "$lsb_img"
+    mkdir -p "$(dirname "$lsb_img")"
+    dd if=/dev/zero of="$lsb_img" bs=1M count=8
+    mke2fs -F "$lsb_img" >/dev/null
+    for d in /bin /etc /usr /usr/bin /sbin; do
+      debugfs -w -R "mkdir $d" "$lsb_img" >/dev/null
+    done
+    tmpfile=$(mktemp)
+    cat <<'EOF' > "$tmpfile"
```

Shellcheck (0.11.0)

[error] Couldn't parse this here document. Fix to allow more checks.

(SC1073)